src/providers/delimitedtext/qgsdelimitedtextfeatureiterator.cpp

```diff
@@ -528,6 +545,28 @@ void QgsDelimitedTextFeatureIterator::fetchAttribute( QgsFeature &feature, int f
         val = QVariant( mSource->mFields.at( fieldIdx ).type() );
       break;
     }
+    case QVariant::LongLong:
+    {
+      if ( ! value.isEmpty() )
+      {
+        // Use std library because QString( "9189304972279762602").toLongLong( &ok )
```

Comment:
Would be good to determine if this IS a current qt bug, because we could/should get KDAB to investigate!

Reply:
I filed a bug: https://bugreports.qt.io/browse/QTBUG-98725

Reply:
It must have been a local issue: I cannot reproduce on another machine with the exact same libraries. I've reverted to use QString::toLongLong.